### ISSUE TITLE
Parse error on pluralization rule

### ISSUE THREAD

**Issue by dandax** [angular/vscode-ng-language-service#45]
I'm getting the following error when using a pluralization rule:

```
[Angular]
Parse Error: Missing expected : at the end of the expression [
    {fieldCount, plural, =0 {no fields} =1 {1 field} other {{{fieldCount}} fields}}
] in @ 4:45
```

My code is as follows

**field-list.component.ts**
```
...
    fieldCount: number = 3;
...
```

**field-list.component.html**
```
<div>
    {fieldCount, plural, =0 {no fields} =1 {1 field} other {{{fieldCount}} fields}}
</div>
```

Note: Angular parses the expression perfectly fine and displays as expected in the browser.

**Comment by chuckjaz** [comment]
Tracking in https://github.com/angular/angular/issues/15625

**Comment by chuckjaz** [comment]
This is fixed in `0.1.2`
